### ISSUE TITLE
About `skmodel.PortfolioSelection`

### ISSUE THREAD

**Issue by Mamba413** [abess-team/skscope#14]
1. Replace `seed` with `random_state`? The latter is a convention parameter in `scikit-learn`.
2. Replace `hist` with `cov_matrix `?  See this: https://pyportfolioopt.readthedocs.io/en/latest/_modules/pypfopt/efficient_frontier/efficient_frontier.html#EfficientFrontier.__init__
3. Replace `k` with `s` might be better because we use `s` in the other places when illustrating sparsity constraint optimization. 
4. Replace `lambda_` with `lambda`?
5. Have you tested whether it is really compatible with `scikit-learn`? For example, you may test whether `sklearn.model_selection.GridSearchCV` can be used for selecting `lambda`?
